druid-shell/src/platform/windows/menu.rs

```diff
@@ -108,4 +125,51 @@ impl Menu {
             AppendMenuW(self.hmenu, MF_SEPARATOR, 0, null());
         }
     }
+
+    /// Get the accels table
+    pub fn accels(&self) -> Option<Vec<ACCEL>> {
+        if self.accels.is_empty() {
+            return None;
+        }
+        Some(self.accels.values().cloned().collect())
+    }
+}
+
+fn convert_hotkey(id: u32, key: &HotKey) -> Option<ACCEL> {
+    let mut virt_key = FVIRTKEY;
+    let key_mods: KeyModifiers = key.mods.into();
+    if key_mods.ctrl {
+        virt_key |= FCONTROL;
+    }
+    if key_mods.alt {
+        virt_key |= FALT;
+    }
+    if key_mods.shift {
+        virt_key |= FSHIFT;
+    }
+
+    let raw_key = match key.key {
+        KeyCompare::Code(code) => code.to_i32()?,
+        KeyCompare::Text(text) => {
+            // See https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-vkkeyscana
+            let code = unsafe { VkKeyScanA(text.chars().next().unwrap() as i8) };
```

Comment:
I'm really not sure about this.

It might be fine. But it certainly could be incorrect; the cast to `i8` will be lossy for many possible `char`s. 

What values do we expect  to find here? Does this text have to be ascii? 

I think it's probably worth being explicit and defensive, unless we have a better idea what's going on; so I would A) not unwrap, and B) verify that the char is ascii before calling `VkKeyScanA`. I might also consider logging an error if it isn't?

Reply:
Agree, and actually I should use should use `VkKeyScanW` and `encode_utf8` instead too :)